staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -324,6 +327,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 		}
 
 		if err := watchHandler(start, w, r.store, r.expectedType, r.expectedGVK, r.name, r.expectedTypeName, r.setLastSyncResourceVersion, r.clock, resyncerrc, stopCh); err != nil {
+			retry.After(err)
```

Comment:
I consciously suggested using:
```
err := watchHandler(...)
retryAfter(err)
if err != nil {
   ...
}
```

Because I want to record nil errors here too (if they appear).